perf/pinot/playlist.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<include>run/testenv.mk</include>
+	<test>
+		<testCaseName>pinot-BenchmarkDictionaryCreation</testCaseName>
+		<command>export JAVA_OPTS=$(JVM_OPTIONS); cd $(Q)$(TEST_RESROOT)$(D)pinot/pinot-perf/target/pinot-perf-pkg/bin$(Q); bash $(Q)$(TEST_RESROOT)$(D)pinot/pinot-perf-pkg/bin/pinot-BenchmarkDictionaryCreation.sh$(Q)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>perf</group>
+		</groups>
+		<vendors>
+			<vendor>microsoft</vendor>
+		</vendors>
+	</test>
+	<test>
+		<testCaseName>pinot-BenchmarkDictionaryCreation</testCaseName>
+		<command>export JAVA_OPTS=$(JVM_OPTIONS); cd $(Q)$(TEST_RESROOT)$(D)pinot/pinot-perf/target/pinot-perf-pkg/bin$(Q); bash $(Q)$(TEST_RESROOT)$(D)pinot/pinot-perf-pkg/bin/pinot-BenchmarkDictionaryCreation.sh$(Q)</command>
+		<levels>
+			<level>sanity</level>
```

Comment:
What's the difference between level=extended and level=sanity?

Reply:
sanity testing in AQA tests is a quick and shallow type of testing that focuses on verifying the basic functionality of the Adoptium build, while extended testing is a more comprehensive and thorough type of testing that includes additional types of testing to uncover more subtle defects in the Adoptium build.